### PR TITLE
Grammar and wording tweaks on homepage

### PR DESCRIPTION
**Pull request by Nezteb**
I noticed a few places on the homepage with some minor grammar issues and also changed some of the wording. Let me know if anything seems incorrect or out-of-place!

### CODE REVIEW

**Comment by Nezteb** [comment]
I don't have permissions to view why the checks failed on Netlify. @yhatt

File: website/components/top/Description.tsx [hunk 1]
```diff
@@ -14,12 +14,12 @@ export const Description = ({ example }: DescriptionProps) => {
   return (
     <section className="container mx-auto py-16">
       <h2 className="w-5/6 mx-auto text-gradient text-center text-3xl font-bold md:text-4xl">
-        The great experience to create slide deck with Markdown
+        Easily create beautiful slide decks with Markdown
```
Comment (yhatt) [inline website/components/top/Description.tsx:17]: For differentiation against other slide tools, we would like to keep using the word "experience" in a lead sentence.

File: website/components/top/Features.tsx [hunk 2]
```diff
@@ -186,29 +186,29 @@ const cards = [
...
     >
-      Marp family has the rich toolset to assist your work.{' '}
+      The Marp ecosystem contains a rich toolset to assist your work.{' '}
       <a
         href="https://marketplace.visualstudio.com/items?itemName=marp-team.marp-vscode"
         rel="noopener noreferrer"
         target="_blank"
       >
-        <b>Marp for VS Code</b>
+        <b>The Marp for VS Code extension</b>
```
Comment (yhatt) [inline website/components/top/Features.tsx:195]: We are treating "Marp for VS Code" as a proper noun to emphasize, so this change feels like to unnatural.

File: website/components/top/Features.tsx [hunk 3]
```diff
@@ -186,29 +186,29 @@ const cards = [
...
       </a>{' '}
-      extension can preview editting Markdown and custom theme immediately.{' '}
+      allows you to edit and preview slide Markdown and custom theming in your editor.{' '}
       <a
         href="https://github.com/marp-team/marp-cli/"
         rel="noopener noreferrer"
         target="_blank"
       >
-        <b>Marp CLI</b>
+        <b>The Marp CLI</b>
```
Comment (yhatt) [inline website/components/top/Features.tsx:203]: The same case as "Marp for VS Code".

File: website/components/top/GetStarted.tsx [hunk 4]
```diff
@@ -244,9 +244,9 @@ export const GetStarted = () => (
           </Button>
         </Card>
         <Card
-          description="Marpit, independented from Marp, is the skinny framework to transform Markdown + CSS theme to the deck composed of HTML + CSS. It has designed to output only minimum assets."
+          description="Marpit (independented from Marp) is the framework that transforms Markdown and CSS themes to slide decks composed of HTML/CSS. It is optimized to output only the minimum set of assets required."
           name="Marpit framework"
-          summary="The skinny framework for creating slide deck from Markdown"
+          summary="The lightweight framework for creating slide decks from Markdown"
```
Comment (yhatt) [inline website/components/top/GetStarted.tsx:249]: "The skinny framework" represents the concept of Marpit framework and we want not to change this. I think "lightweight" cannot describe the advantage of the framework (minimum outputs).

**Comment by Nezteb** [comment]
@yhatt Fixed!

**Comment by yhatt** [comment]
Thank you so much, @Nezteb!

If possible, could you check grammar of a description metadata? The first sentence of the metadata is based on a lead, so may require updating the description too.

https://github.com/marp-team/marp/blob/c39ae80a20514a75f17c0c1efbe5173dc56f82ea/website/components/Layout.tsx#L18-L19

**Comment by Nezteb** [comment]
@yhatt Alright, I made the two descriptions consistent!

**Review by yhatt — APPROVED** [review]
Looks great. Thank you for your contribution!

**Pull request merged**